rllib/algorithms/impala/impala.py

```diff
@@ -687,7 +784,10 @@ def get_samples_from_workers(
                     timeout_seconds=self._timeout_s_sampler_manager,
                     return_obj_refs=return_object_refs,
                 )
-            elif self.workers.local_worker() and self.config.create_env_on_local_worker:
+            elif self.workers.local_worker() and (
```

Comment:
need a way to sample even if sampling from local worker.

Reply:
It was, its just there was never a reason to run impala with 0 workers before this, and definitely would hang if there were 0 workers.